### DISCUSSION TITLE
Getting (cid:0) text while reading pdf which is not present in the pdf

### DISCUSSION THREAD

**Discussion by sarveshkrg** [jsvine/pdfplumber#818] [Ask for help with specific PDFs]
While reading below attached pdf **page number 13** using extract_text() method, I am getting **(cid:0)** but this text is not present in the pdf file.

I need help to know why this text **(cid:0)** is coming and how to avoid it in case it happens for another pdfs also.

Input file
[00058054.PDF](https://github.com/jsvine/pdfplumber/files/10777050/00058054.PDF)
Page number **13**

**Comment by jsvine** [comment]
Hi @sarveshkrg, the `(cid:#)` texts come from the PDF missing a mapping of the font characters to unicode. See this thread (and [others](https://github.com/pdfminer/pdfminer.six/search?q=%22cid:%22&type=issues)) in the `pdfminer.six` (`pdfplumber`'s main dependency) repository for more details / potential fixes: https://github.com/pdfminer/pdfminer.six/issues/368
